cloudformation/external-slurmdbd/external_slurmdbd/external_slurmdbd_stack.py

```diff
@@ -391,6 +415,20 @@ def _add_cloudwatch_log_group(self):
             retention=logs.RetentionDays.ONE_WEEK,
         )
 
+    def _add_s3_bucket(self):
+        return s3.CfnBucket(
+            self,
+            id="ExternalSlurmdbdS3Bucket",
+            bucket_name=self.stack_name.lower(),
+            public_access_block_configuration=s3.CfnBucket.PublicAccessBlockConfigurationProperty(
+                block_public_acls=True,
+                block_public_policy=True,
+                ignore_public_acls=True,
+                restrict_public_buckets=True,
+            ),
+            versioning_configuration=s3.CfnBucket.VersioningConfigurationProperty(status="Enabled"),
```

Comment:
Do we need versioning ?
If the answer is yes we may want to consider some "expiration rules" to delete versions older than `<something>`.

Reply:
The bucket will contain only 2 or 3 files, which will seldom be updated. I believe there's no need to worry about that.